packages/integrations/image/README.md

```diff
@@ -90,6 +90,10 @@ import { Image, Picture } from '@astrojs/image/components';
 
 The included `sharp` transformer supports resizing images and encoding them to different image formats. Third-party image services will be able to add support for custom transformations as well (ex: `blur`, `filter`, `rotate`, etc).
 
+Astro's `<Image />` and `<Picture />` components will throw an error if no `alt` text is provided. 
+
+If the image is merely decorative (i.e. doesn't contribute to the understanding of the page), set `alt=""` so it is ignored by screen readers.
+
```

Comment:
LOVE this new feature, @DerYeger! I have installed some Chrome extensions myself to make sure I'm not allowed to Tweet without alt-text on my images.  😄 

Please ALSO check the "Usage" heading further down the page where the attributes, and whether or not they are required are listed for both components. There should be an `alt` heading and description for each one of them. It looks like it's currently not there for `<Image />` but is there for `<Picture />` though only saying "if it's provided."

These should both prominently display the new info, and double check all the examples to make sure they all show `alt=` on every component. 

Lastly, @DerYeger  & @tony-navillus, what exactly *is* the error message that is thrown? (Sorry, hard to look through everything on my phone here) If it's not, can it be an explicit message to add alt text rather than a generic error?

Thanks!

Reply:
Hi, I used the suggestion of https://github.com/withastro/astro/issues/4291#issuecomment-1215949055 as the error message, replacing `image` with `picture` for the respective component.

Reply:
Great! I've merged the PR for the docs "Images Guide" page just now. Just let me know if you're up for making the above edits under "Usage" (as an attribute, like all the other `aspectRatio`, `src` etc.) in addition to your note here, or whether you'd like me to take care of that!  But, it DOES need to be there as an attribute for each component.

Reply:
These changes don't appear to have been made, so I'm making them now here!